prov/efa/src/efa_ep.c

```diff
@@ -611,6 +633,132 @@ static struct fi_ops efa_ep_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static void efa_ep_progress_internal(struct efa_cq *efa_cq, uint64_t flags)
+{
+	struct util_cq *cq = &efa_cq->util_cq;
+	int cq_entries = 500;
```

Comment:
Can we make this a macro and define it up front in the file (and make it a tunable param)?